tmpl/confirm_email.mustache

```diff
@@ -26,7 +26,7 @@
         <form id="form" action="/confirm" method="get">
           <input type="hidden" name="session" value="{{ session_id }}">
           <div class="entry">
-            <input type="text" name="code" maxlength="6" autofocus>
+            <input type="text" name="code" maxlength="8" autofocus>
```

Comment:
Should we allow a longer length here, and then trim whitespace on the backend? Might help people who copy-and-paste the code, but accidentally select an extra space on either side.